torch/nn/modules/module.py

```diff
@@ -491,6 +492,7 @@ def register_forward_pre_hook(self, hook):
 
             hook(module, input) -> None or modified input
 
+        The input contains only the positional arguments given to the module.
```

Comment:
Can you get the kwargs? If not, maybe be explicit? What about a positional argument specified as a kwarg?

Reply:
You get everything captured by `*args` on the function call. Not sure if that contains positional arguments passed as kwargs?

Reply:
After testing, there is no nothing of positional as kwarg :D If you do `foo=2` when calling a function, it is a kwarg.

Reply:
Updating to explicitly state that the kwargs are ignored.